spec/markdown_spec.rb

```diff
@@ -33,6 +33,8 @@ def category_exists?(title)
           expect(frontmatter).to be
 
           aggregate_failures do
+            expect(frontmatter['description']).to be
```

Comment:
empty string would still pass this, right? like I could be a maliciously compliant page author with:

```
description: ""
```

```suggestion
            expect(frontmatter['description']).to_not be_empty
```